marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.queries.tsx

```diff
@@ -73,6 +73,13 @@ describe('save/manage queries scenarios, developer role', () => {
         browsePage.getSelectedQueryDescription().should('contain', 'new-query-2 description');
         browsePage.getTableCell(1, 2).should('contain','102');
         browsePage.getTableCell(2, 2).should('contain','103');
+
+        // Verify discard/apply icons are not shown after applying query from zero state page.
```

Comment:
@saarimrahman there is already apply query scenario in the tests, please update them to check for icons visibility